vmdk_plugin/utils/refcount/refcnt.go

```diff
@@ -241,8 +241,13 @@ func (r RefCountsMap) Decr(vol string) (uint, error) {
 	}
 
 	rc.count--
+
+	if rc.count < 0 {
+		log.Warning("Decr: refcnt is negative deleting,
```

Comment:
this message does not make sense to me , can you rephrase ?  Something like `"Decr: internal error - refcnt is negative. Trying to recover - deleting the counter. name = ..., refct=...")...`.